.devcontainer/devcontainer.json

```diff
@@ -18,13 +19,20 @@
         "vscode": {
             // Add the IDs of extensions you want installed when the container is created.
             "extensions": [
-                "ms-dotnettools.csharp",
+                "github.vscode-github-actions",
+                "GitHub.copilot",
+                "GitHub.vscode-github-actions",
```

Comment:
```suggestion
                "github.vscode-github-actions",
                "GitHub.copilot",
                "GitHub.vscode-github-actions",
```

Two entries here that differ only on case - does VSCode care?

Reply:
Good question. I don't think so?